src/parsec_client/operations/psa_asymmetric_decrypt.md

```diff
@@ -0,0 +1,38 @@
+# PsaAsymmetricDecrypt
+
+Decrypt a short message with a private key. Opcode: 9 (`0x0009`)
+
+## Parameters
+
+| Name       | Type                                                                      | Description                                                                                                           |
+|------------|---------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| `key_name` | String                                                                    | Name of the key to use for the operation                                   |
+| `alg`      | [`AsymmetricEncryption`](psa_algorithm.md#asymmetricencryption-algorithm) | An asymmetric encryption algorithm that is compatible with the type of key |
+| `message`  | Vector of bytes                                                           | Short message to decrypt                                                   |
+| `salt`     | Vector of bytes                                                           | Salt to use during encryption, if supported by the algorithm     |
+
+- `key_name` must be the name of an RSA asymmetric key pair or private key. The key must allow the [usage
+   flag](psa_key_attributes.md#usageflags-type) `decrypt`.
+- `salt` can be provided if supported by the algorithm. If the algorithm does not support salt, pass NULL. If the 
+algorithm supports optional salt, pass NULL to indicate no salt. For RSA PKCS#1 v1.5 encryption, no salt is supported.
```

Comment:
The references to `NULL` should probably be replaced with `empty vector` or something like that.

Reply:
or nothing at all (as in remove the sentence)? Because I guess it will depend how the optional field is implemented in the target language. For example in Rust, it is `None` I think.

Reply:
It also depends on the contract type - it might be like this in protobuf, maybe CBOR would be different. I said empty vector because the field is declared as vector above